layouts/_default/list.algolia.json

```diff
@@ -0,0 +1,54 @@
+{{/* Generates a valid Algolia search index */}}
+{{- $hits := slice -}}
+{{- $section := $.Site.GetPage "section" .Section }}
+{{- $validVars := $.Param "algolia.vars" | default slice -}}
+{{- $validParams := $.Param "algolia.params" | default slice -}}
+{{/* Include What Pages? */}}
+{{ range $i, $hit := .Site.AllPages }}
```

Comment:
I'm assuming that this only goes through the Hugo generated pages, not the Antora generated ones, am I correct?

Reply:
yes, it is for now :)

Reply:
I've added antora indices using docker as per in this guide https://gitlab.com/antora/antora-ui-default/issues/44#note_68053321